weblate/templates/component.html

```diff
@@ -1,10 +1,22 @@
 {% extends "base.html" %}
 {% load i18n %}
+{% load static %}
 {% load permissions %}
 {% load translations %}
+{% load compress %}
 {% load crispy_forms_tags %}
 {% load metrics %}
 
+{% block extra_script %}
+  {% compress js %}
+    <script defer data-cfasync="false" src="{% static 'vendor/moment.js' %}{{ cache_param }}"></script>
+    <script defer data-cfasync="false" src="{% static 'vendor/daterangepicker.js' %}{{ cache_param }}"></script>
+  {% endcompress %}
+  {% compress css %}
+    <link rel="stylesheet" href="{% static 'vendor/daterangepicker.css' %}{{ cache_param }}" />
+  {% endcompress %}
+{% endblock %}
+
```

Comment:
I'm still thinking whether including this only in selected templates is the way to go. Having it on the dashboard/project/component page makes it likely that every user will load this anyway. With upcoming https://github.com/WeblateOrg/weblate/issues/11846 even more pages will load this. On the other side, I've not looked at the performance hit caused by loading the additional JavaScript and CSS. In case it's measurable, let's keep loading separate (but let's make it reusable instead of copying the same code to every template). Otherwise, I'd just include it in the base template to make the maintenance simpler.

Reply:
Since the usage of this feature is growing across pages, it is more appropriate to transfer its loading in the base template. Their links is now transferred to the `base.html` for easier reuse and maintenance.